tests/integration/targets/test_azure_manage_networking_stack/tasks/main.yaml

```diff
@@ -0,0 +1,161 @@
+---
+- name: Test role cloud.azure_ops.azure_manage_networking_stack with default options
+  block:
+    # Create Networking Stack
+    - name: Create Networking stack
+      ansible.builtin.include_role:
+        name: cloud.azure_ops.azure_manage_networking_stack
+      vars:
+        azure_manage_networking_stack_operation: create
+   
+    # Verify Networking stack created
+    - name: Get Virtual network info
+      azure.azcollection.azure_rm_virtualnetwork_info:
+        resource_group: "{{ azure_manage_networking_stack_resource_group }}"
+        name: "{{ azure_manage_networking_stack_virtual_network }}"
+      register: vnet_info
+
+    - name: Get subnet info
+      azure.azcollection.azure_rm_subnet_info:
+        resource_group: "{{ azure_manage_networking_stack_resource_group }}"
+        name: "{{ azure_manage_networking_stack_subnet }}"
+        virtual_network_name: "{{ azure_manage_networking_stack_virtual_network }}"
+      register: subnet_info
+
+    - name: Ensure Networking stack created
+      ansible.builtin.assert:
+        that:
+          - vnet_info.virtualnetworks | length == 1
+          - subnet_info.subnets | length == 1
+
+    # Delete Networking stack
+    - name: Delete Networking stack
+      ansible.builtin.include_role:
+        name: cloud.azure_ops.azure_manage_networking_stack
+      vars:
+        azure_manage_networking_stack_operation: delete
+
+    # Verify Networking stack deleted
+    - name: Get Resource group
+      azure.azcollection.azure_rm_resourcegroup_info:
+        name: "{{ azure_manage_networking_stack_resource_group }}"
+      register: rg_info
+
+    - name: Ensure Resource group deleted
+      ansible.builtin.assert:
+        that:
+          - rg_info.resourcegroups | length == 0
+        fail_msg: "{{ rg_info.resourcegroups }}"
+
+  always:
+    - name: Delete a Resource group including resources it contains
+      azure_rm_resourcegroup:
+        name: "{{ azure_manage_networking_stack_resource_group }}"
+        force_delete_nonempty: true
+        state: absent
+
+- name: Test non-default delete/create option for cloud.azure_ops.azure_manage_networking_stack role
+  block:
+    # Create Resource group
+    - name: Create Resource group
+      azure.azcollection.azure_rm_resourcegroup:
+        name: "{{ azure_manage_networking_stack_resource_group }}"
+        location: "{{ azure_manage_networking_stack_region }}"
+
+    # Create Networking Stack
+    - name: Create Networking stack
+      ansible.builtin.include_role:
+        name: cloud.azure_ops.azure_manage_networking_stack
```

Comment:
I don't see any difference between this and task 6.  Same vars.  Am I missing something?

Reply:
In the first test (line6) resource group is created during the creation of the networking stack
In the second test (line68) the resource group was created in advance and during the creation of the networking stack this step was skipped